src/Umbraco.Core/Services/IContentService.cs

```diff
@@ -295,12 +290,6 @@ IContent CreateBlueprintFromContent(IContent blueprint, string name, int userId
     /// </summary>
     OperationResult Save(IContent content, int? userId = null, ContentScheduleCollection? contentSchedule = null);
 
-    /// <summary>
-    ///     Saves documents.
-    /// </summary>
-    // TODO: why only 1 result not 1 per content?!
-    OperationResult Save(IEnumerable<IContent> contents, int userId = Constants.Security.SuperUserId);
```

Comment:
Actually, doesn't look like this can be removed.  On the base class the return type is different - `Attempt<OperationResult?>`.

Reply:
I have added it back in with a "new" in front to indicate that it's intentional, this will also remove the build warning and shouldn't be a breaking change. I also updated some more XML document comments in the IContentService file.